index.html

```diff
@@ -8,44 +8,273 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <div id="header-background">
-        <header id="flex-nav">
-            <ul>
-                <li id="logo"><a href="index.html">My Logo</a></li>
-                <nav id="burguer-menu-icon ">
-                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
-                </nav>
-            </ul>
-        </header>
-        <section id="primary-text">
-            <h1 class="primary-text-header">I'm Enoque <br/> Glad to see<br/>you!</h1>
-            <p id="hello-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <p id="connect">LET'S CONNECT</p>
-            <ul id="social">
-                <li>
-                    <a href="#">
-                        <img src="./img/twitter.png" alt="twitter link">
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        <img src="./img/Linkedin.png" alt="Linkedin link">
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        <img src="./img/m.png" alt="m link">
-                    </a>
+    <div class="site-wrapper">
+        <section id="header-background">
+            <header id="flex-nav">
+                <ul>
+                    <li id="logo"><a href="index.html">My Logo</a></li>
+                    <nav id="burguer-menu-icon ">
+                        <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                    </nav>
+                </ul>
+            </header>
+            <div id="primary-text">
+                <h1 class="primary-text-header">I'm Enoque <br/> Glad to see<br/>you!</h1>
+                <p id="hello-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p id="connect">LET'S CONNECT</p>
+                <ul id="social">
+                    <li>
+                        <a href="#">
+                            <img src="./img/twitter.png" alt="twitter link">
+                        </a>
+                    </li>
+                    <li>
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents for readability. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)